src/main.js

```diff
@@ -332,8 +332,16 @@ function grammarsFromScriptElements(optNodeOrNodeList) {
   return ns;
 }
 
-function makeRecipe(recipeFn) {
-  return recipeFn.call(new Builder());
+function makeRecipe(recipe) {
+  if (typeof recipe === 'function') {
+    return recipe.call(new Builder());
+  } else {
```

Comment:
I will leave it in. Currently, semantics recipes and the bootstrapped JS-files (dist/built-in-rules.js, etc.) use "JSO"(N) with `makeRecipe`. It's faster - since it does not have to be deserialized - and it also circumvents escaping quotes (lots of strings in strings if you put JSON in a string first).
For pure grammar recipes, this doesn't matter and they can be read and processed as strings.